src/main/java/com/bpodgursky/jbool_expressions/ExprUtil.java

```diff
@@ -49,9 +49,9 @@
 
   public static <K> Expression<K> stripNegation(Or<K> or, And<K> internalAnd, Expression<K> omitFromAnd){
     Expression<K>[] childrenNew = ExprUtil.allExceptMatch(or.expressions, internalAnd);
-    List<Expression<K>> newChildren = Lists.newArrayList(childrenNew);
+    List<Expression<K>> newChildren = new ArrayList<>(Arrays.asList(childrenNew));
```

Comment:
Couldn't this just be 

````
List<Expression<K>> newChildren = Arrays.asList(childrenNew);
````
and avoid a copy?

Reply:
Oh weird, it's returning a different fixed size ArrayList.  That is annoying

Reply:
nvm, ok, I get it.  it's just a view over the array.  so it's only one copy.  ignore this whole chain.